UseCase50/composer.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "neighborhoods/kojo-fitness",
+  "type": "library",
+  "description": "Neighborhoods Kōjō Examples provides example patterns for implementing Kōjō workers.",
+  "license": "MIT",
+  "keywords": [],
+  "authors": [
+    {
+      "name": "Brad Wilson",
+      "email": "brad.wilson@neighborhoods.com"
+    }
+  ],
+  "config": {
+    "sort-packages": true
+  },
+  "require": {
+    "php": ">=7.2",
+    "neighborhoods/kojo": "5.x-dev",
+    "symfony/console": "4.3.*"
```

Comment:
Symfony Console 4.4 was causing a problem, this is just a workaround (kojo proper will fix it itself)